recipes/perl-extutils-makemaker/meta.yaml

```diff
@@ -0,0 +1,37 @@
+
+{% set name = "perl-extutils-makemaker" %}
+{% set version = "6.66" %}
+{% set sha256 = "0abaed8963d825ed2ca52785250f37c8c9f42d0810c9246252c06825955c37f1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://cpan.metacpan.org/authors/id/B/BI/BINGOS/ExtUtils-MakeMaker-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl
+
+  run:
+    - perl
+
+test:
+  commands:
+    - perl -e "use ExtUtils::MakeMaker"
```

Comment:
Apparently this isn't a good test, because it's passing with nothing actually installed from the package.